guacamole/src/main/webapp/translations/en.json

```diff
@@ -677,6 +677,7 @@
         "FIELD_HEADER_SFTP_ROOT_DIRECTORY"        : "File browser root directory:",
         "FIELD_HEADER_SFTP_USERNAME"              : "Username:",
         "FIELD_HEADER_SWAP_RED_BLUE"    : "Swap red/blue components:",
+        "FIELD_HEADER_USERNAME"     : "Username:",
```

Comment:
"Username" is common enough that there are likely translations in the other supported languages which can be safely copied without having to be a native speaker. Thoughts on adding those?

Reply:
Definitely - I've shamelessly copied the Telnet protocol version of the translation for each language to the VNC protocol.  Two things I noticed as I was doing this:
* There is some level of inconsistency within the translations as to whether they are organized alphabetically or logically.  I tried to be consistent within the inconsistency - but I can try to correct the inconsistency if you'd rather.
* The Italian translation looks like most of the items probably need to be removed - I don't think they're actually translated.  Again, I just copied it and carried on, but if it's a good time to strip out untranslated strings within that translation, I can go for that...